### PR TITLE
In search(), replace splitlines() with lines array split on \n.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**
splitlines() will split subjects on for example \r, causing parse error.
Only splitting on \n avoid this problem.  Unfortunately this will
cause the code to also consider blank lines, so these need to be ignored.

### CODE REVIEW

**Comment by sebix** [comment]
Could we instead split be `\n`?

**Comment by petterreinholdtsen** [comment]
[Sebastian]
> Could we instead split be `\n`?

I am unable to parse this sentence.  Can you rephrase?

Note, the request response is split on \n in line 506 of the code.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by petterreinholdtsen** [comment]
Note, the failing build is because http://demo.request-tracker.fr:80 is blocking access to the test suite.

**Comment by sebix** [comment]
> Note, the request response is split on \n in line 506 of the code.

Ah, thanks.
> Note, the failing build is because http://demo.request-tracker.fr:80 is blocking access to the test suite.

Which won't change (#28)

**Comment by e3rd** [comment]
Thanks!
